Foundation/NSLocale.swift

```diff
@@ -55,7 +55,7 @@ public class NSLocale : NSObject, NSCopying, NSSecureCoding {
         return copyWithZone(nil)
     }
     
-    public func copyWithZone(_ zone: NSZone) -> AnyObject { NSUnimplemented() }
+    public func copyWithZone(_ zone: NSZone) -> AnyObject { return self }
```

Comment:
The base abstract implementation of NSLocale in objc does exactly this:
```
- (id)copyWithZone:(NSZone *)zone {
    return [self retain];
}
```
So it stands to reason that since Locale's are immutable that it is just a return of self